builtin/logical/ssh/path_roles.go

```diff
@@ -363,6 +364,17 @@ func pathRoles(b *backend) *framework.Path {
 					Name: "Signing Algorithm",
 				},
 			},
+			"not_before_duration": {
+				Type:    framework.TypeDurationSecond,
+				Default: 30,
+				Description: `
+                The duration before now the SSH cert needs to be signed.
```

Comment:
Just for context, if you run `vault path-help ssh/roles` this aught to show up there. I think you'll find that it should probably be on one line, not on two.

Reply:
Also, as a nit, perhaps `The duration before the issuance time that the SSH certificate should be backdated by.` might be a touch clearer?

Reply:
Or maybe `The duration that the SSH certificate should be backdated by at issuance.` ?

Reply:
Makes sense, I took inspiration on the comment of the PR you linked but your descriptions are better.

Reply:
Ah, no, I like the `notBeforeDuration` as it leaves us open to add a `validAfter` field later with an explicit date if we want that (rather than a relative offset; not really sure why we'd want that in general though, maybe for Y10K certs backdated to 1970).